make bazel-release fails with python3

`make bazel-release` fails with the following error

```
dims@rex:~/go/src/k8s.io/kubernetes$ make bazel-release
INFO: Analyzed target //build/release-tars:release-tars (2960 packages loaded, 41335 targets configured).
INFO: Found 1 target...
ERROR: /home/dims/go/src/k8s.io/kubernetes/build/debs/BUILD:197:2: MakeDeb build/debs/kube-proxy__amd64.deb failed (Exit 1) make_deb failed: error executing command
  (cd /home/dims/.cache/bazel/_bazel_dims/8b714d02207463747ec9588937c2cb2f/sandbox/linux-sandbox/10127/execroot/io_k8s_kubernetes && \
  exec env - \
  bazel-out/host/bin/external/bazel_tools/tools/build_defs/pkg/make_deb '--output=bazel-out/k8-fastbuild/bin/build/debs/kube-proxy__amd64.deb' '--changes=bazel-out/k8-fastbuild/bin/build/debs/kube-proxy__amd64.changes' '--data=bazel-out/k8-fastbuild/bin/build/debs/kube-proxy-data-amd64.tar' '--package=kube-proxy' '--architecture=amd64' '--maintainer=Kubernetes Authors <kubernetes-dev+release@googlegroups.com>' '--version=@bazel-out/k8-fastbuild/bin/build/version' '--description=Kubernetes Service Proxy' '--depends=iptables (>= 1.4.21)' '--depends=iproute2' '--homepage=http://kubernetes.io' '--distribution=unstable' '--urgency=medium')
Execution platform: @bazel_tools//platforms:host_platform

Use --sandbox_debug to see verbose messages from the sandbox: make_deb failed: error executing command
  (cd /home/dims/.cache/bazel/_bazel_dims/8b714d02207463747ec9588937c2cb2f/sandbox/linux-sandbox/10127/execroot/io_k8s_kubernetes && \
  exec env - \
  bazel-out/host/bin/external/bazel_tools/tools/build_defs/pkg/make_deb '--output=bazel-out/k8-fastbuild/bin/build/debs/kube-proxy__amd64.deb' '--changes=bazel-out/k8-fastbuild/bin/build/debs/kube-proxy__amd64.changes' '--data=bazel-out/k8-fastbuild/bin/build/debs/kube-proxy-data-amd64.tar' '--package=kube-proxy' '--architecture=amd64' '--maintainer=Kubernetes Authors <kubernetes-dev+release@googlegroups.com>' '--version=@bazel-out/k8-fastbuild/bin/build/version' '--description=Kubernetes Service Proxy' '--depends=iptables (>= 1.4.21)' '--depends=iproute2' '--homepage=http://kubernetes.io' '--distribution=unstable' '--urgency=medium')
Execution platform: @bazel_tools//platforms:host_platform

Use --sandbox_debug to see verbose messages from the sandbox
Traceback (most recent call last):
  File "/home/dims/.cache/bazel/_bazel_dims/8b714d02207463747ec9588937c2cb2f/sandbox/linux-sandbox/10127/execroot/io_k8s_kubernetes/bazel-out/host/bin/external/bazel_tools/tools/build_defs/pkg/make_deb.runfiles/bazel_tools/tools/build_defs/pkg/make_deb.py", line 367, in <module>
    main(FLAGS(sys.argv))
  File "/home/dims/.cache/bazel/_bazel_dims/8b714d02207463747ec9588937c2cb2f/sandbox/linux-sandbox/10127/execroot/io_k8s_kubernetes/bazel-out/host/bin/external/bazel_tools/tools/build_defs/pkg/make_deb.runfiles/bazel_tools/tools/build_defs/pkg/make_deb.py", line 339, in main
    version=GetFlagValue(FLAGS.version),
  File "/home/dims/.cache/bazel/_bazel_dims/8b714d02207463747ec9588937c2cb2f/sandbox/linux-sandbox/10127/execroot/io_k8s_kubernetes/bazel-out/host/bin/external/bazel_tools/tools/build_defs/pkg/make_deb.runfiles/bazel_tools/tools/build_defs/pkg/make_deb.py", line 311, in GetFlagValue
    flagvalue = flagvalue.decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
----------------
Note: The failure of target @bazel_tools//tools/build_defs/pkg:make_deb (with exit code 1) may have been caused by the fact that it is a Python 2 program that was built in the host configuration, which uses Python 3. You can change the host configuration (for the entire build) to instead use Python 2 by setting --host_force_python=PY2.

If this error started occurring in Bazel 0.27 and later, it may be because the Python toolchain now enforces that targets analyzed as PY2 and PY3 run under a Python 2 and Python 3 interpreter, respectively. See https://github.com/bazelbuild/bazel/issues/7899 for more information.
----------------
Target //build/release-tars:release-tars failed to build
INFO: Elapsed time: 109.791s, Critical Path: 64.64s
INFO: 4994 processes: 4994 linux-sandbox.
FAILED: Build did NOT complete successfully
Makefile:638: recipe for target 'bazel-release' failed
make: *** [bazel-release] Error 1
```

--- a/build/root/.bazelrc
+++ b/build/root/.bazelrc
@@ -1,5 +1,8 @@
 startup --expand_configs_in_place
 
+# Force us to use python2
+build --host_force_python=PY2
+
 # Show us information about failures.
 build --verbose_failures
 test --test_output=errors

--- a/build/root/.bazelrc
+++ b/build/root/.bazelrc
@@ -1,6 +1,8 @@
 startup --expand_configs_in_place
 
-# Force us to use python2
+# https://github.com/kubernetes/kubernetes/pull/79807
+# https://github.com/bazelbuild/rules_docker/issues/842
+# https://github.com/bazelbuild/bazel/issues/7899
 build --host_force_python=PY2
 
 # Show us information about failures.

